### PR TITLE
Update jsbeautify to 1.6.2 

### PR DESCRIPTION
**Pull request by mathieug**
Update jsbeautify to 1.6.2 and its dependency six to 1.10.0

### CODE REVIEW

File: js_formatter.py [hunk 1]
```diff
@@ -34,20 +32,13 @@ def _try_get_short_path(path):
 # crazyness to get jsbeautifier.unpackers to actually import
 # with sublime's weird hackery of the path and module loading
 add_lib_path(libs_path)
-add_lib_path(os.path.join(libs_path, "six-1.8.0"))
-add_lib_path(os.path.join(libs_path, "jsbeautifier"))
-add_lib_path(os.path.join(libs_path, "jsbeautifier", "jsbeautifier"))
```
Comment (jdc0589) [inline js_formatter.py]: @mathieug I remember a lot of pain early on in the project getting this stuff to work, and a lot of explicit includes seemed necessary. I am by no means good at python, but have you tested this pretty thoroughly across sublime versions and operating systems?

Comment (mathieug) [inline js_formatter.py]: I've tested on Mac/Win with ST3. The thing is, I've kept `add_lib_path(libs_path)` and this `libs/` folder only contains modules (all its folders contain __init__.py). It's why it works perfectly.

Comment (jdc0589) [inline js_formatter.py]: I'm gonna need to test a bit more. I feel like I remember doing that originally, but having to add all the explicit stuff to get it working on all platforms; which I realize seems crazy and totally un-necessary.

Comment (mathieug) [inline js_formatter.py]: six was not a module, I've renamed it : libs/{six-1.8.0/six.py → six/__init__.py}
jsbeautify was not a module : libs/jsbeautifier/{jsbeautifier → }/__init__.py

Comment (mathieug) [inline js_formatter.py]: It's why you needed the add_libs_path for both of them.

**Comment by atif089** [comment]
Hi @mathieug,

Thanks for pushing this. I really can't wait until this pull request is merged. If I want to merge these changes into local copy of my plugin, can you please tell me how can I do that?

Thanks

**Comment by mathieug** [comment]
Sure @atif089.
Download a zip from my repository (where the fork is), extract it into your Packages folder then start Sublime Text.
**Be sure Sublime Text is not open when you add stuff in this folder.**

**Pull request merged**